js/apps/account-ui/src/root/Root.tsx

```diff
@@ -49,6 +49,10 @@ export const Root = () => {
     [t]
   );
 
+  const brandImage = environment.logo ? environment.logo : "logo.svg";
+  const brandOnClickHandler = () =>
+    window.location.replace(environment.logoUrl);
```

Comment:
@edewit Doing this as an `onClick` doesn't turn out quite right.  The cursor doesn't show up as a clickable link when you hover over the logo.  Maybe we can fix this in the Masthead project?

Reply:
yes created a PR https://github.com/keycloak/keycloak/pull/20596